<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D-конструктор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resourses.qrc">
              <normaloff>:/images/icons/4x2B.png</normaloff>:/images/icons/4x2B.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resourses.qrc">
              <normaloff>:/images/icons/4x2L.png</normaloff>:/images/icons/4x2L.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resourses.qrc">
              <normaloff>:/images/icons/myModel.png</normaloff>:/images/icons/myModel.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MainGLWidget" name="openGLWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Поворот:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shiftXPos">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrow-Circle-icon.png</normaloff>:/images/icons/Arrow-Circle-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shiftXNeg">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrow-Circle-icon2.png</normaloff>:/images/icons/Arrow-Circle-icon2.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shiftYPos">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrow-Circle-icon.png</normaloff>:/images/icons/Arrow-Circle-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shiftYNeg">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrow-Circle-icon2.png</normaloff>:/images/icons/Arrow-Circle-icon2.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>z</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shiftZPos">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrow-Circle-icon.png</normaloff>:/images/icons/Arrow-Circle-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="shiftZNeg">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrow-Circle-icon2.png</normaloff>:/images/icons/Arrow-Circle-icon2.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Сдвиг:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="turnXPos">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrows-Down-Left-icon.png</normaloff>:/images/icons/Arrows-Down-Left-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="turnXNeg">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrows-Up-Right-icon.png</normaloff>:/images/icons/Arrows-Up-Right-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="turnYPos">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrows-Down-Left-icon.png</normaloff>:/images/icons/Arrows-Down-Left-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="turnYNeg">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrows-Up-Right-icon.png</normaloff>:/images/icons/Arrows-Up-Right-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>z</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="turnZPos">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrows-Down-Left-icon.png</normaloff>:/images/icons/Arrows-Down-Left-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="turnZNeg">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resourses.qrc">
                <normaloff>:/images/icons/Arrows-Up-Right-icon.png</normaloff>:/images/icons/Arrows-Up-Right-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">mainglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
